pageserver/src/virtual_file/owned_buffers_io/io_buf_aligned.rs

```diff
@@ -0,0 +1,11 @@
+#![allow(unused)]
```

Comment:
i think it's now being used in layer readers?

Reply:
Yep: removed in 4b4a3edb804d265347a5ead9acbed4f72e499ed4